applications/nrf5340_audio/overlay-mcuboot.conf

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_PCD_APP=y
+CONFIG_BOARD_ENABLE_CPUNET=n
+
+# Those kconfig should be moved from board file to prj.conf
```

Comment:
Shall we remove this note?

Reply:
Maybe rephrase here?
I'd like to make a note here which kconfigs should be moved from board config to prj.
When it's time for board config cleanup ,we have reference here.